pinot-common/src/main/java/com/linkedin/pinot/common/utils/StringUtil.java

```diff
@@ -15,12 +15,14 @@
  */
 package com.linkedin.pinot.common.utils;
 
+import java.io.UnsupportedEncodingException;
 import javax.annotation.Nonnull;
 import org.apache.commons.lang.StringUtils;
 
 
 public class StringUtil {
   private static final char NULL_CHARACTER = '\0';
+  private static String charSet = "UTF-8";
```

Comment:
Please make this private static **final** with name **UTF-8**

Reply:
meant to do that.

Reply:
I prefer change the variable name to "UTF_8" because based on the method name, it has to be UTF-8 instead of any other char set. and also the convention for private static final variable should be all capital letters.

Reply:
woops, missed this last comment of yours. Will fix in the next code checkin. Sorry.